src/messages/MOSDMap.h

```diff
@@ -27,6 +27,7 @@ class MOSDMap : public Message {
 
  public:
   uuid_d fsid;
+  uint64_t encode_features;
```

Comment:
please assign a default value to `encode_features`.

Reply:
fixed